doc/api/http2.md

```diff
@@ -3150,13 +3150,16 @@ will result in a [`TypeError`][] being thrown.
 added: v8.4.0
 -->
 * `headers` {HTTP/2 Headers Object} An object describing the headers
-* `callback` {Function}
-
-Call [`http2stream.pushStream()`][] with the given headers, and wraps the
-given newly created [`Http2Stream`] on `Http2ServerResponse`.
+* `callback` {Function} Called once `http2stream.pushStream()` is finished,
+or either when the attempt to create the pushed Http2Stream has failed or has
+been rejected, or the state of `Http2ServerRequest` is closed.
+  * `err` {Error}
+  * `stream` {ServerHttp2Stream} The newly-created `ServerHttp2Stream` object.
```

Comment:
Consistency nit: it seems we do not add periods if the description is not a full sentence (see `headers` description above).

Reply:
OK